en/docs/services/create-your-first-service.md

```diff
@@ -1,177 +1,250 @@
+6. Now let’s extract the current total cases count from the response and store it in a variable.
+
+    To do this, click the last **+** icon in the low-code diagram and click **Variable**.
     
-        ![Update Existing Connection](../assets/img/services/update-existing-connection.png){.cInlineImage-half}
-        
-    4. Click **API Calls** and then click the existing connection.
+    Enter information as follows:
     
-        ![Select Existing connection](../assets/img/services/select-existing-connection.png){.cInlineImage-half}
-        
-    5. Enter the details as follows:
-        
-        | **Field**               | **Value**                       |
-        |-------------------------|---------------------------------|
-        | **OPERATION**           | **get**                         |
-        | **Resource Path**       | `"/total/country/united-states"`|
-        | **Select Payload Type** | **JSON**                        |
-        
-        !!! info
-            To select **JSON** as the payload type, you must enable **Do you want to extract a payload?**.
-            ![Do you want to extract a payload button](../assets/img/services/enable-payload.png){.cInlineImage-half}
+    | **Field**      | **Value**                      |
+    |----------------|--------------------------------|
+    | **Type**       | `var`                          |
+    | **Name**       | `totalCases`                   |
+    | **Expression** | `statusByCountry?.cases ?: 0d` |
+    
+    Save the information.
+    
+## Step 3: Get the population data
 
-    6. Click **Save & Done**.
-        
-   Now you have designed the service to fetch the active COVID-19 cases in the United States.
+To get the population data:
+
+1. Click the last **+** icon in the low-code diagram and click **API Calls**.
+
+2. Click on **World Bank API** in the available connection list.
+
+3. In the **Connection Name** field, enter `worldBankClient`, and save.
+
+4. Click the last **+** icon in the low-code diagram and click **API Calls**.
+
+5. Under **Choose existing connection**, click **worldBankClient**.
+
+6. In the **Operation** field, select **Get Country Population**. Then enter the following information in the other fields that appear.
+
+    1. In the **Country Code** field, enter `country`.
     
-4. To store the response from the public API to your service, follow this procedure:
+    2. In the **Date** field, enter `date`.
+    
+    3. Click **Optional**, and in the **Format** field, enter `json`.
+
+    4. In the **Response Variable Name** field, enter `populationByCountry`
 
-    1. Click the last **+** icon in your low-code diagram.
+    Save the information.
 
-        ![Add New Statement](../assets/img/services/add-custom-statement.png){.cInlineImage-half}
+7. Now let’s extract the population from the response and store it in a variable.
+
+    To do this, click the last **+** icon in the low-code diagram and click **Variable**.
```

Comment:
"In the **Statements** tab" part is missing

Reply:
Did not mention it because the **Statements** tab is open by defaults and this statement type is visible as soon as the dialog box opens.